docs/writings/web-apps/part-1.md

```diff
@@ -0,0 +1,199 @@
+### Blog Post Series: Understanding Web Applications in oCIS
+
+#### Part 1: Overview and How to Load Extensions
+
+---
+
+**Introduction to Web Applications in oCIS**
+
+In today's fast-paced digital world, web applications play a crucial role in enhancing user experience and functionality.
+
+For organizations using ownCloud Infinite Scale (oCIS), understanding how to leverage web applications can significantly enhance their productivity and user engagement.
+
+In this series, we will delve into the world of web applications in oCIS, focusing on how extensions are loaded and utilized.
+
+---
+
+**Why We Need Web Applications**
+
+Web applications are essential for several reasons:
+
+1. **Accessibility**:\
+Web applications can be accessed from any device with an internet connection, providing flexibility and convenience.
+2. **Customization**:\
+They allow organizations to customize and extend the functionality of their platform to meet specific needs.
+3. **Integration**:\
+Web applications enable seamless integration with third-party services and tools, enhancing overall functionality.
+
+By leveraging web applications, organizations can create a more tailored and efficient environment for their users, ensuring that they have the tools they need to succeed.
+
```

Comment:
Honestly, I think the two chapters before are not needed.  I would really suggest to keep that short by saying something like
"Infinite Scale comes with  a world class web interface to manage file resources, but it can be extended by utilizing oCIS as a construction set for custom web apps"